getting-started/codecs.md

```diff
@@ -42,26 +41,35 @@ Tremor's internal data representation is JSON-like. The supported value types ar
 
 ### Codecs
 
-Tremor neither requires nor validates schemas and works with schemaless or unstructured
-data. Validation can be asserted with the tremor-script language. `Onramps`, `Offramps`
-and other components of tremor may however require or expect conformance with schemas.
+Tremor neither requires nor validates schemas and works with schemaless or unstructured data. Validation can be asserted with the tremor-script language. `Onramps`, `Offramps`, and other components of Tremor may, however, require or expect conformance with schemas.
 
-For specific components their documentation should be consulted for correct usage.
+For specific components, their documentation should be consulted for correct usage.
 
 Tremor supports the encoding and decoding of the following formats:
 
+<<<<<<< HEAD
```

Comment:
Another merge artifact, I think we want to keep the relative path instead of the full one, since all pages are now the same